### PR TITLE
Examining an ore box you are adjacent to shows its contents.

### PR DESCRIPTION
**Pull request by harmonyn**
Why this is a good addition:

1. when I'm mining I often want to know how much ore I have without having to tetris my inventory
2. Other servers do it this way and I keep forgetting

### CODE REVIEW

File: code/modules/mining/satchel_ore_boxdm.dm [hunk 1]
```diff
@@ -28,6 +28,10 @@
 	else
 		return ..()
 
+/obj/structure/ore_box/examine(mob/user)
+	if(Adjacent(user))
+		show_contents(user)
```
Comment (Cyberboss) [inline code/modules/mining/satchel_ore_boxdm.dm]: call ..()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mining/satchel_ore_boxdm.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/mining/satchel_ore_boxdm.dm]: . = ..()

Comment (bgobandit) [inline code/modules/mining/satchel_ore_boxdm.dm]: done

File: code/modules/mining/satchel_ore_boxdm.dm [hunk 2]
```diff
@@ -28,13 +28,18 @@
...
+
 /obj/structure/ore_box/attack_hand(mob/user)
 	if(Adjacent(user))
 		show_contents(user)
 
 /obj/structure/ore_box/attack_robot(mob/user)
 	if(Adjacent(user))
 		show_contents(user)
+	. = ..()
```
Comment (ChangelingRain) [inline code/modules/mining/satchel_ore_boxdm.dm]: not here!

File: code/modules/mining/satchel_ore_boxdm.dm (hunk 1 above)
Comment (bgobandit) [inline code/modules/mining/satchel_ore_boxdm.dm]: this is done, idk why it doesn't show up

**Comment by AnturK** [comment]
From quick glance looks like ghosts will be able to use these to empty them.

File: code/modules/mining/satchel_ore_boxdm.dm [hunk 3]
```diff
@@ -28,13 +28,19 @@
...
+
 /obj/structure/ore_box/attack_hand(mob/user)
 	if(Adjacent(user))
 		show_contents(user)
 
 /obj/structure/ore_box/attack_robot(mob/user)
 	if(Adjacent(user))
 		show_contents(user)
+	..()
```
Comment (lzimann) [inline code/modules/mining/satchel_ore_boxdm.dm]: remove this

**Comment by lzimann** [comment]
@bgobandit

**Pull request merged**